### PR TITLE
Read device confirmation button + improve text

### PR DESCRIPTION
**Pull request by llakssz**
Checks to see if the confirmation button is O or X - and uses it.
Before, the 'accept' button was hardcoded to O.
My vita uses O too, but it is not good for users who use X.

Text in the program is improved/made more uniform.
Things like grammar, capitalization etc...

Just changes to make it more comfortable! :shoe:

Thanks for making the program!

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 TITLE_ID = SAVEMGR00
-TITLE = Save Manager
+TITLE = Vita Save Manager
```
Comment (d3m3vilurr) [inline Makefile]: hum... i dont need `Vita` prefix.
This project always run on the vita or pstv ;)

File: main.c [hunk 2]
```diff
@@ -24,6 +26,20 @@ uint32_t white = RGBA8(0xFF, 0xFF, 0xFF, 0xFF);
...
+char enter_text[8];
+char cancel_text[8]; 
+
+char* concat(const char *s1, const char *s2)
+{
+    char *result = malloc(strlen(s1)+strlen(s2)+1);
+    strcpy(result, s1);
+    strcat(result, s2);
+    return result;
+}
```
Comment (d3m3vilurr) [inline main.c]: this function can cause memory leak.

File: main.c [hunk 3]
```diff
@@ -511,6 +529,23 @@ int main() {
     sceAppMgrAppParamGetString(0, 9, title , 256);
     sceAppMgrAppParamGetString(0, 12, titleid , 256);
 
+    SceAppUtilInitParam init_param;
```
Comment (d3m3vilurr) [inline main.c]: please use `SceAppUtilInitParam init_param = {0};`

File: main.c (hunk 2 above)
Comment (llakssz) [inline main.c]: I don't like C and I hate 'c strings' - I found that online and, ok - how can I fix it?

File: Makefile (hunk 1 above)
Comment (llakssz) [inline Makefile]: Ok, I thought that was the name of the app, no problem.

File: main.c (hunk 3 above)
Comment (llakssz) [inline main.c]: Ok

File: main.c (hunk 2 above)
Comment (d3m3vilurr) [inline main.c]: i also dislike c string functions :)
please use snprintf instead malloc & strcpy.

**Pull request merged**